docs/articles/baseplatform/running-qemu-smoketests.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+.. _running-qemu-smoketests:
+
+Running smoke tests on QEMU
+===========================
+
+To run smoke tests on QEMU target, proceed as follows,
+
+.. code-block:: bash
+
+	# Bitbake core-image-pelux-minimal-dev for qemu-x86-64-nogfx variant
```

Comment:
s/variant/target/

Reply:
qemu-x86-64-nogfx seems to be a variant of QEMU target.

Reply:
it's a target platform, isn't it?

Reply:
variants are minimal ant qtauto

Reply:
Fixed